rio/src/RIO/Prelude/Exit.hs

```diff
@@ -0,0 +1,15 @@
+module RIO.Prelude.Exit
+  ( module System.Exit
```

Comment:
I prefer having explicit export lists to avoid accidentally leaking functionality added in later versions of base. Can you either:

1. Make this list explicit
2. Make the import list below explicit